### PR TITLE
Max width, and truncation with ellipsis, for group names in enrichments tab

### PR DESCRIPTION
**Pull request by adamabeshouse**
Fixes one entry of https://github.com/cBioPortal/cbioportal/issues/6031

### CODE REVIEW

File: src/shared/components/Checkbox.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+export interface ICheckboxProps {
+    value?:string;
+    checked:boolean;
+    onClick:()=>void;
+    label:any;
+}
+
+@observer
+export default class Checkbox extends React.Component<ICheckboxProps, {}> {
```
Comment (kalletlak) [inline src/shared/components/Checkbox.tsx]: there is already `react-bootstrap` checkbox. how different is this from that?

Comment (alisman) [reply src/shared/components/Checkbox.tsx]: agreed.  if this is different, name should describe what it is.

Comment (adamabeshouse) [reply src/shared/components/Checkbox.tsx]: the difference is the flex alignment, should I change the name to FlexAlignedCheckbox

Comment (kalletlak) [reply src/shared/components/Checkbox.tsx]: I think even that does it right

**Pull request merged**